### ISSUE TITLE
Use local user directory for creating the site cache

### ISSUE THREAD

**Issue by okkhoy** [MarkBind/markbind-cli#24]
At this moment, `markbind` tries to use the cache in the install location.
This fails on Linux (and I suspect on Mac as well)
Instead, use the user's home directory to create the cache.

```
info: Cloning git@github.com:nus-cs2103-AY1718S2/website.git into ../../../../../usr/lib/node_modules/markbind-cli/node_modules/gh-pages/.cache
error: fatal: could not create work tree dir '../../../../../usr/lib/node_modules/markbind-cli/node_modules/gh-pages/.cache': Permission denied
```

**Comment by acjh** [comment]
When does this happen? I've not encountered this on a Mac.

**Comment by okkhoy** [comment]
When I try to deploy

**Comment by Gisonrg** [comment]
The location is handled by a library we used, `gh-pages`. We should take a look into this.

**Comment by nicholaschuayunzhi** [comment]
I believe this is the relevant function:

https://github.com/tschaub/gh-pages/blob/ee95b051ad84c8a37e1fb10cb418fa96b95419cd/lib/index.js#L11-L13

```js
function getCacheDir() {
  return path.relative(process.cwd(), path.resolve(__dirname, '../.cache'));
}
```

**Comment by acjh** [comment]
Fixed in [v1.6.0](https://github.com/MarkBind/markbind-cli/releases/tag/v1.6.0).
